tests/test_bytes_subclass_support.py

```diff
@@ -63,19 +64,19 @@ def base64url_to_memoryview(data: str) -> memoryview:
         verification = verify_authentication_response(
             credential=AuthenticationCredential(
                 id="fq9Nj0nS24B5y6Pkw_h3-9GEAEA3-0LBPxE2zvTdLjDqtSeCSNYFe9VMRueSpAZxT3YDc6L1lWXdQNwI-sVNYrefEcRR1Nsb_0jpHE955WEtFud2xxZg3MvoLMxHLet63i5tajd1fHtP7I-00D6cehM8ZWlLp2T3s9lfZgVIFcA",
-                raw_id=base64url_to_memoryview(
+                raw_id=
                     "fq9Nj0nS24B5y6Pkw_h3-9GEAEA3-0LBPxE2zvTdLjDqtSeCSNYFe9VMRueSpAZxT3YDc6L1lWXdQNwI-sVNYrefEcRR1Nsb_0jpHE955WEtFud2xxZg3MvoLMxHLet63i5tajd1fHtP7I-00D6cehM8ZWlLp2T3s9lfZgVIFcA"
-                ),
+                ,
                 response=AuthenticatorAssertionResponse(
-                    authenticator_data=base64url_to_memoryview(
+                    authenticator_data=
                         "SZYN5YgOjGh0NBcPZHZgW4_krrmihjLHmVzzuoMdl2MBAAAABw"
-                    ),
-                    client_data_json=base64url_to_memoryview(
+                    ,
+                    client_data_json=
                         "eyJ0eXBlIjoid2ViYXV0aG4uZ2V0IiwiY2hhbGxlbmdlIjoiZVo0ZWVBM080ank1Rkl6cURhU0o2SkROR3UwYkJjNXpJMURqUV9rTHNvMVdOcWtHNms1bUNZZjFkdFFoVlVpQldaV2xaa3pSNU1GZWVXQ3BKUlVOWHciLCJvcmlnaW4iOiJodHRwOi8vbG9jYWxob3N0OjUwMDAiLCJjcm9zc09yaWdpbiI6ZmFsc2V9"
-                    ),
-                    signature=base64url_to_memoryview(
+                    ,
+                    signature=
                         "RRWV8mYDRvK7YdQgdtZD4pJ2dh1D_IWZ_D6jsZo6FHJBoenbj0CVT5nA20vUzlRhN4R6dOEUHmUwP1F8eRBhBg"
-                    ),
+                    ,
```

Comment:
I don't understand why this test was changed. `base64url_to_memoryview()` emulates an RP that chooses to use a different data structure than plain `bytes`, and then decides to map their data to the shape of `AuthenticationCredential`...

Maybe what's missing here is that `AuthenticationCredential` is an instance of [the spec's `PublicKeyCredential` structure](https://www.w3.org/TR/webauthn-2/#iface-pkcredential), which says `rawId` is an `ArrayBuffer` - that's captured in this library as `raw_id: bytes`. If you want to make it possible for `AuthenticationCredential.raw_id` to be a `str` then I don't think that makes sense, since the library will ensure that `raw_id` bytes can be base64url-encoded to be the same value as `id`. If an RP wants to use something else to store `raw_id` bytes in their database prior to the bytes ever making it to `webauthn.verify_*_response()`, then that should be up to them. They just need to get it back to bytes before this library's methods are called.